### PR TITLE
Break down models

### PR DESCRIPTION
**Pull request by TMcMeans**
I did my best to break down the models into their specified files. When reviewing, please let me know if certain functions belong in other areas. Tons of commits here, most are separated by specific model refactors. Let me know!

### CODE REVIEW

**Comment by TMcMeans** [comment]
I ran the entire test suite a couple times before committing but the e3e tests are being super brittle. Sorry about that!

File: apps/api/models/provider/domain.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const fromCliniko = contact => ({
```
Comment (lionsdwarf) [inline apps/api/models/provider/domain.js]: You nailed it here! 👏 These are exactly the kinds of functions that should live in the Domain file.

File: apps/api/models/provider/mapper.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import * as cliniko from 'cliniko'
+
+export const findById = id => cliniko.findOne({ resource: 'contacts', id })
```
Comment (lionsdwarf) [inline apps/api/models/provider/mapper.js]: Again, well done with this Mapper function!

**Comment by TMcMeans** [comment]
Okay, tell my what ya'll think about how I broke down the content-module into better gql/domain/mapper function distinctions. I have some concerns about the child modules function being in the wrong place but it essentially does similar logic to the new findByName function that I wrote to pull out the data transaction. I *think* I did this right??

File: apps/api/index.js [hunk 3]
```diff
@@ -4,12 +4,12 @@
  */
 import { ApolloServer, gql } from 'apollo-server-express'
 import Client from './models/client'
-import * as ContentModule from './models/content-module'
+import ContentModule from './models/content-module'
 import Coupon from './models/coupon'
 import * as log from 'log'
-import * as Provider from './models/provider'
-import * as Salve from './models/salve'
-import * as SalveCategory from './models/salve-category'
```
Comment (craigspaeth) [inline apps/api/index.js]: Goodbye asterisks 🤘

File: apps/api/models/content-module/index.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import * as Gql from './gql'
+import * as Mapper from './mapper'
+import * as Domain from './domain'
+
+/**
```
Comment (craigspaeth) [inline apps/api/models/content-module/index.js]: Nitpick: Usually, these bigger module-level block comments are put at the very top above imports.

File: apps/api/models/salve/domain.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      momentTz
+        .tz(timestamp, 'MMM DD, YYYY hh:mm:ss A', 'America/Los_Angeles')
+        .isAfter(moment(now).subtract(JOB_INTERVAL_L, 'milliseconds'))
+    )
+  )
+  return salves.map((salve, i) => ({ salve, results: recentResults[i] }))
+}
+
+// Reorder salves according to order of Person.salves
+export const orderedSalves = (salves, orderedIds) =>
```
Comment (craigspaeth) [inline apps/api/models/salve/domain.js]: 😍 So nice to see these cleanly co-located now. The function names `recentResultsWithSalves`, `orderedSalves`, `applyNotify` all sound so clearly like "business logic" only relevant to the Octave "domain". Feels good!

File: apps/api/models/salve/gql.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  queries: `
+      salve(
+        id: ID!
+      ): Salve
+      salves: [Salve]
+    `
+}
+
+/**
+ * -----------------------------------------------------------------------------
```
Comment (craigspaeth) [inline apps/api/models/salve/gql.js]: Nitpick: We could probably kill these big divider comments now that things are split into modules.

File: apps/api/test/models/cliniko-user.test.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import cliniko from 'cliniko'
+import ClinikoUser from '../../models/cliniko-user'
+
+jest.mock('cliniko', () => ({
+  find: jest.fn()
+}))
+
+test('findAll returns all users in Cliniko', async () => {
```
Comment (craigspaeth) [inline apps/api/test/models/cliniko-user.test.js]: Thanks!

**Comment by craigspaeth** [comment]
👏 👏 👏 Beautiful work! I only have nitpicks you can feel free to ignore. Looks great! This is good to merge once the merge conflict is resolved.

File: apps/api/models/content-module/index.js [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+ *     h1: ["Welcome to Octave", "Step 1"],
+ *     p: ["Let's get started"],
+ *     a: ["Breath in"],
+ *     images: ["placekitten.com/100/200"]
+ *   }
+ *   ```
+ *
+ */
+
+export default {
```
Comment (craigspaeth) [inline apps/api/models/content-module/index.js]: Excellent job breaking down ContentModule into the right pieces!

File: apps/api/index.js [hunk 9]
```diff
@@ -3,12 +3,14 @@
  * namespace powered by Apollo Server
  */
 import { ApolloServer, gql } from 'apollo-server-express'
+import Client from './models/client'
```
Comment (craigspaeth) [inline apps/api/index.js]: This is where the linter is complaining.

**Pull request merged**